control-plane/subcommand/server-acl-init/command_ent_test.go

```diff
@@ -100,9 +119,15 @@ func TestRun_ConnectInject_SingleDestinationNamespace(t *testing.T) {
 			actRule, _, err := consul.ACL().BindingRuleRead(rules[0].ID, namespaceQuery)
```

Comment:
```suggestion
			aclRule, _, err := consul.ACL().BindingRuleRead(rules[0].ID, namespaceQuery)
```
super nit, non blocking